cegs_portal/search/admin.py

```diff
@@ -16,7 +23,73 @@ class DNAFeatureAdmin(admin.ModelAdmin):
 admin.site.register(DNAFeature, DNAFeatureAdmin)
 
 
+class OtherFileForm(forms.ModelForm):
```

Comment:
This seems unnecessary right now -- It doesn't do anything as far as I can tell. I think you should remove it.